app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -76,6 +77,16 @@
     private BroadcastReceiver mReceiver;
 
     /**
+     * Spellcheck related fields
+     */
+    private int autoSpellcheck;
+    private final int spellcheckEnabledType = InputType.TYPE_CLASS_TEXT |
+                                              InputType.TYPE_TEXT_FLAG_AUTO_CORRECT;
+
+    private final int spellcheckDisabledType = InputType.TYPE_CLASS_TEXT |
+                                               InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS;
```

Comment:
Swiftkey displays suggestions even with this flag, we'll need to use the VISIBLE_PASSWORD flag.